main.go

```diff
@@ -54,6 +55,6 @@ func main() {
 		<-sig
 		os.Exit(0)
 	}()
-	program.StartSession(ip, protocol.OTRProtocol{})
+	program.StartSession(ip, new(protocol.OTRProtocol))
```

Comment:
@wavyllama Let's just leave it like this for now, the other PR fixes this all anyways.